dlclive/dlclive.py

```diff
@@ -478,3 +486,154 @@ def close(self):
         self.is_initialized = False
         if self.display is not None:
             self.display.destroy()
+
+
+class MultiAnimalDLCLive(DLCLive):
+    def __init__(
+        self,
+        model_path,
+        n_animals,
+        n_multibodyparts,
+        track_method: str = "box",
+        min_hits: int = 1,
+        max_age: int = 1,
+        sim_threshold: float = 0.6,
+        resize: Optional[float] = None,
+        convert2rgb: bool = True,
+        processor: Optional['Processor'] = None,
+        display: typing.Union[bool, Display] = False,
+        pcutoff: float = 0.5,
+        display_radius: int = 3,
+        display_cmap: str = "bmy",
+    ):
+        if track_method not in ("box", "ellipse"):
+            raise ValueError("`track_method` should be either `box` or `ellipse`.")
+
+        self.model_path = model_path
+        super().__init__(
+            Path(model_path).parent,
+            resize=resize,
+            convert2rgb=convert2rgb,
+            processor=processor,
+            display=display,
+            pcutoff=pcutoff,
+            display_radius=display_radius,
+            display_cmap=display_cmap,
+        )
+        self.n_animals = n_animals
+        self.n_multibodyparts = n_multibodyparts
+        self.track_method = track_method
+        self.min_hits = min_hits
+        self.max_age = max_age
+        self.sim_threshold = sim_threshold
+
+    def read_config(self):
+        cfg_path = Path(self.path).resolve() / "pose_cfg.yaml"
+        if not cfg_path.exists():
+            raise FileNotFoundError(
+                f"The pose configuration file for the exported model at {str(cfg_path)} was not found. Please check the path to the exported model directory"
+            )
+
+        self.cfg = load_config(cfg_path)
+        self.cfg["batch_size"] = 1
+        self.cfg["init_weights"] = self.model_path.split(".")[0]
+        self.identity_only = self.cfg["num_idchannel"] > 0
+
+    def init_inference(
+        self,
+        frame=None,
+        allow_growth=False,
+        **kwargs,
+    ):
+        self.sess, self.inputs, self.outputs = predict.setup_pose_prediction(
+            self.cfg, allow_growth=allow_growth,
+        )
+
+        if self.track_method == "box":
+            self.mot_tracker = trackingutils.SORTBox(
```

Comment:
attribute not declared in `__init__`. would be nice to just make this a property